torchtune/training/_distributed.py

```diff
@@ -584,4 +587,4 @@ def shard_model(
             fully_shard(m, **fsdp_kwargs)
 
     # Finally shard the entire model to account for any stragglers
-    fully_shard(model)
```

Comment:
this seems to be dropping `**fsdp_kwargs` by accident? it prevents cpu_offloading for root model

Reply:
Yeah I had meant to fix this, thanks for adding it here!